src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -544,11 +544,10 @@ boolean canRepair(
 
               String msg = "Postponed due to affected hosts already doing repairs";
               repairRunner.updateLastEvent(msg);
-              handlePotentialStuckRepairs(busyHosts, metrics.getNode());
               return false;
             }
           }
-        } catch (InterruptedException | ExecutionException | ConcurrentException e) {
+        } catch (InterruptedException | ExecutionException e) {
```

Comment:
If we don't catch a ConcurrentException here, then it'll presumably fall through. Is the function above the only way that exception can be thrown?

Reply:
Yep, that's the only function in the block which threw the exception